src/test/kotlin/org/jetbrains/gradle/ext/BuildIdeArtifactTest.kt

```diff
@@ -244,11 +250,19 @@ class BuildIdeArtifactTest {
   }
 
   fun createTestArchive(sourceDir: File, destinationFile: File) {
-    myProject.tasks.create("testArchive", Zip::class.java) {
-      it.from(sourceDir)
-      it.destinationDir = destinationFile.parentFile
-      it.archiveName = destinationFile.name
-    }.execute()
+    var out = ZipOutputStream(BufferedOutputStream(FileOutputStream(destinationFile)))
+
+    sourceDir.walkBottomUp().forEach { file ->
+      if(!file.isDirectory()) {
+        var fi = FileInputStream(file)
```

Comment:
Can probably use `Files.copy` for this one as well.

Reply:
done